src/bridging/providers/across/AcrossApi.ts

```diff
@@ -0,0 +1,371 @@
+   *
+   * If this value isn't provided in the request, the API will assume the latest block timestamp on mainnet.
+   *
+   * Example: 1653547649
+   */
+  timestamp?: number
+
+  /**
+   * Optionally override the relayer address used to simulate the fillRelay() call that estimates the gas costs
+   * needed to fill a deposit. This simulation result impacts the returned suggested-fees. The reason to customize the
+   * EOA would be primarily if the recipientAddress is a contract and requires a certain relayer to submit the fill,
+   * or if one specific relayer has the necessary token balance to make the fill.
+   *
+   * Example: 0x428AB2BA90Eba0a4Be7aF34C9Ac451ab061AC010
+   */
+  relayer?: string
+}
+
+export interface SuggestedFeesLimits {
+  /**
+   * The minimum deposit size in the tokens' units.
+   *
+   * Note: USDC has 6 decimals, so this value would be the number of USDC multiplied by 1e6. For WETH, that would be 1e18.
+   */
+  minDeposit: string
+
+  /**
+   * The maximum deposit size in the tokens' units. Note: The formatting of this number is the same as minDeposit.
+   */
+  maxDeposit: string
+
+  /**
+   * The max deposit size that can be relayed "instantly" on the destination chain.
+   *
+   * Instantly means that there is relayer capital readily available and that a relayer is expected to relay within
+   * seconds to 5 minutes of the deposit.
+   */
+  maxDepositInstant: string
+
+  /**
+   * The max deposit size that can be relayed with a "short delay" on the destination chain.
+   *
+   * This means that there is relayer capital available on mainnet and that a relayer will immediately begin moving
+   * that capital over the canonical bridge to relay the deposit. Depending on the chain, the time for this can vary.
+   *
+   * Polygon is the worst case where it can take between 20 and 35 minutes for the relayer to receive the funds
+   * and relay.
+   *
+   * Arbitrum is much faster, with a range between 5 and 15 minutes. Note: if the transfer size is greater than this,
+   * the estimate should be between 2-4 hours for a slow relay to be processed from the mainnet pool.
+   */
+  maxDepositShortDelay: string
+
+  /**
+   * The recommended deposit size that can be relayed "instantly" on the destination chain.
+   *
+   * Instantly means that there is relayer capital readily available and that a relayer is expected to relay
+   * within seconds to 5 minutes of the deposit. Value is in the smallest unit of the respective token.
+   */
+  recommendedDepositInstant: string
+}
+
+export interface SuggestedFeesResponse {
+  /**
+   * Percentage of the transfer amount that should go to the relayer as a fee in total. The value is inclusive of lpFee.pct.
+   *
+   * This is the strongly recommended minimum value to ensure a relayer will perform the transfer under the current
+   * network conditions.
+   *
+   * The value returned in this field is guaranteed to be at least 0.03% in order to meet minimum relayer fee requirements
```

Comment:
Where does the `0.03%` minimum come from?

Reply:
same closing